src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/ItemDetails.js

```diff
@@ -150,7 +133,7 @@ export const ItemDetails = ({
       label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.section`} />}
     >
       <Row left="xs">
-        <Col xs={12}>
+        <Col xs={12}>-
```

Comment:
Is the `-` a typo?

Reply:
Hi @BogdanDenis Yes, sorry, I'll remove this one